test/jdk/java/awt/color/ICC_Profile/ValidateICCHeaderData/ValidateICCHeaderData.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 8337703
+ * @bug 8337703 8347377 8358057
  * @summary To verify if ICC_Profile's setData() and getInstance() methods
```

Comment:
I have added 8347377 where this test was added and 8358057 for the current issue. @honkar-jdk I cannot access the 8337703 is it a "closed bug" or it is a typo and we can drop that id?